E2e multiline comments everywhere, with minor breaking changes

Droogans opened the pull request:
This started as a chore to clean up comments, but it ended up becoming more. I realized that my editor makes it easy to make multiline comments by creating many single-line comments, and that this is a bit hard on the eyes. I'm switching over to multiline comments because they look better.

Also, since I submitted a pull request with breaking changes in it earlier this week, I thought I might as well get as many of them as I can for the minor version change that the first will cause.

parlarjb commented on src/rxApp/rxApp.page.js:
```diff
@@ -4,37 +4,23 @@ var Page = require('astrolabe').Page;
-
     title: {
-        get: function () { return this.lblNavTitle.getText(); }
+        get: function () { return this.rootElement.$('.site-title').getText(); }
```
What's your thinking behind this change?

Droogans replied:
The commit message is a bit buried in the changeset...here's the relevant snippet:

> Only use element properties on a page object if the element is used in more than one place in the functions exposed by the page object. In cases where this is only one reference to a button, link, etc., just use an inline element selector.

parlarjb commented:
Can you make sure the final squashed commit message contains a "BREAKING CHANGE" footer, as described here? https://docs.google.com/document/d/1QrDFcIiPjSLDn3EL15IJygNPiHORgU1_OOAqWjiDU5Y/edit#heading=h.gbbngquhe0qa

This will ensure that the auto-generated change log makes mention of the breaking change.

Droogans commented:
@parlarjb I made it `BREAKING CHANGES` (plural). Is that going to be seen by the change log generator?

parlarjb commented:
I don't think so. https://github.com/ajoslin/conventional-changelog/blob/master/lib/git.js#L97

Droogans commented:
@parlarjb I just changed it to two separate `BREAKING CHANGE:` paragraphs in one commit, and on second reading of the source code you linked to, I don't think it's going to catch that either.

All in all, I think it's fine. The end result is that one commit will have one breaking change, which is the way it should be, really. I'll update that commit and split it up.

parlarjb commented:
LGTM
